FrostyPlugin/IO/FrostyModWriter.cs

```diff
@@ -275,12 +275,27 @@ public void WriteProject(FrostyProject project)
             // @todo: superbundles
 
             // added bundles
+            bool addedBundlesAllowed = Config.Get<bool>("AddedBundlesFbmod", true);
```

Comment:
where do u add this config boolean?

Reply:
It’s just a setting that can be tweaked by a plugin. Pretty much useless without it so I don’t think it’s necessary to make it a full option in Frosty. Just have it so the plugin can access that value

Reply:
ah okay